test/minikube/001-kube.bats

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bats
+#
+# Tests of podman kube commands with minikube
+#
+
+load ../system/helpers.bash
+
+function setup(){
+    # only set up the minikube cluster before the first test
+    if [ "$BATS_TEST_NUMBER" -eq 1 ]; then
+        minikube start
+        echo $output >&2
+    fi
+    # wait for a bit so all the services in the cluster come up
+    sleep 10
+}
+
+###############################################################################
+# BEGIN tests
+
+@test "minikube - check cluster is up" {
+    run minikube kubectl get nodes
+    [ "$status" -eq 0 ]
+    [[ "$output" == *"Ready"* ]]
+    run minikube kubectl get pods
+    [ "$status" -eq 0 ]
+    [[ "$output" == "No resources found in default namespace." ]]
+}
+
+@test "minikube - deploy generated container yaml to minikube" {
+    cname="test-ctr"
```

Comment:
Consider using a randomly generated container name *and* verifying it doesn't already exist.

Reply:
The name that comes with a randomly generated name is not supported by k8s, they have some specifications around what the name of the container/pod can be.

Reply:
okay, gotcha. I think collecting the minikube log may be good enough coverage for any name-clash problems.